packages/fhir-client/src/components/PatientListView/index.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { Col, Button, Descriptions } from 'antd';
+import { CloseOutlined } from '@ant-design/icons';
+import { useQuery } from 'react-query';
+import {
+  BrokenPage,
+  FHIRServiceClass,
+  ResourceDetails,
+  SingleKeyNestedValue,
+  useSearchParams,
+  viewDetailsQuery,
+} from '@opensrp/react-utils';
+import { LIST_PATIENTS_URL, patientResourceType } from '../../constants';
+import { useTranslation } from '../../mls';
+import { IPatient } from '@smile-cdr/fhirts/dist/FHIR-R4/interfaces/IPatient';
+import { getPatientName } from '../PatientsList/utils';
+import { Link } from 'react-router-dom';
+
+export interface PatientListViewProps {
+  fhirBaseURL: string;
+}
+
+const PatientListView = (props: PatientListViewProps) => {
+  const { fhirBaseURL } = props;
+  const { t } = useTranslation();
+  const { removeParam, sParams } = useSearchParams();
+  const patientId = sParams.get(viewDetailsQuery) ?? undefined;
+
+  const { data, error } = useQuery({
+    queryKey: [patientResourceType, patientId],
+    queryFn: async () =>
+      new FHIRServiceClass<IPatient>(fhirBaseURL, patientResourceType).read(
+        patientId as unknown as string
+      ),
+    enabled: !!patientId,
+  });
+
+  if (error && !data) {
+    return <BrokenPage errorMessage={(error as Error).message} />;
+  }
+
+  if (!patientId || !data) {
+    return null;
+  }
```

Comment:
We always want to render something if an error occurred or data was invalid in a way that will break the component we should communicate that instead of returning nothing.

Reply:
`ResourceNotFound` component